### PR TITLE
Fix pdf download

### PR DESCRIPTION
**Pull request by solivier**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Yes
| New feature?         | No
| BC breaks?           | No
| CI currently passes? | Yes
| Tests pass?          | Yes
| Scenarios pass?      | Yes
| Checkstyle issues?*  | Np
| PMD issues?**        | No
| Changelog updated?   | No
| Fixed tickets        | -
| DB schema updated?   | No
| Migration script?    | No
| Doc PR               | -

### CODE REVIEW

File: spec/Pim/Bundle/PdfGeneratorBundle/Renderer/ProductPdfRendererSpec.php [hunk 1]
```diff
@@ -31,13 +32,14 @@ function it_filters_not_compatible_entities(Category $printer)
...
+        $templating,
         ProductInterface $blender,
         AttributeGroupInterface $design,
-        AttributeInterface $color,
-        $templating
+        AttributeInterface $color
     ) {
         $blender->getAttributes()->willReturn([$color]);
 
+        $path = realpath(__DIR__.'/../../../../../features/Context/fixtures/');
```
Comment (nidup) [inline spec/Pim/Bundle/PdfGeneratorBundle/Renderer/ProductPdfRendererSpec.php]: do you need to put real path here ?

File: spec/Pim/Bundle/PdfGeneratorBundle/Renderer/ProductPdfRendererSpec.php [hunk 2]
```diff
@@ -49,18 +51,24 @@ function it_renders_a_product_without_images(
...
 
     function it_renders_a_product_with_an_image(
+        $templating,
         ProductInterface $blender,
         AttributeGroupInterface $media,
-        AttributeInterface $mainImage,
-        $templating
+        AttributeInterface $mainImage
     ) {
+        $path = realpath(__DIR__ . '/../../../../../features/Context/fixtures/');
```
Comment (nidup) [inline spec/Pim/Bundle/PdfGeneratorBundle/Renderer/ProductPdfRendererSpec.php]: same question here

**Comment by aRn0D** [comment]
Is it related to #2012?

**Comment by solivier** [comment]
@aRn0D Yes it is didn't see the ticket sorry

**Comment by aRn0D** [comment]
@solivier  cool !

**Comment by aRn0D** [comment]
:v:

**Pull request merged**